xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/filters/FilterList.vue

```diff
@@ -93,7 +93,11 @@ export default {
             this.removeFilter();
           } else if (value !== this.value) {
             // If the selected value has an empty value, then use the empty filter, otherwise use the contains filter.
-            this.applyFilter(value, value === '' ? 'empty' : 'contains');
+            if (value === '') {
+              this.applyFilter(value, 'empty');
```

Comment:
Note that here we assume there will always be an empty operator in any LD source for the list type. We need to document this.